src/compiler/crystal/syntax/to_s.cr

```diff
@@ -3,9 +3,9 @@ require "./visitor"
 
 module Crystal
   class ASTNode
-    def inspect(io : IO) : Nil
-      to_s(io)
-    end
+    # def inspect(io : IO) : Nil
+    #   to_s(io)
+    # end
```

Comment:
Can we simply remove these lines? 🙏

Reply:
What would happen in this case?
```
macro node(n)
  {{puts n}}
end

node a + b
```

It is not a so important feature, but sometimes I use it to debug

Reply:
I don't think `inspect` is used in that case?

Reply:
All right, makes sense.

The spec seem to fail, but I have no idea about the reason